testsettings.py

```diff
@@ -1,21 +1,15 @@
 import os
 import dj_database_url
 
+
+DATABASE_URL = os.environ.get("DATABASE_URL", "sqlite:///:memory:")
```

Comment:
one too many `/` here?

Reply:
Weirdly this is actually correct. See https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#connect-strings for an explanation.

Reply:
ok cool :)